test/bpf-loader.test.js

```diff
@@ -0,0 +1,34 @@
+// @flow
```

Comment:
can you add a `test/bin/noop_c.txt` file that describes where this blob came from and how it was generated, to help out the next person that tries to debug why it's failing in the future (like when we make the next entrypoint ABI change)